articles/active-directory/verifiable-credentials/plan-verification-solution.md

```diff
@@ -253,7 +253,7 @@ You can use information in presented VCs to build a user profile. If you want to
 
    * Define a mechanism to deprovision the user profile from the application. Due to the decentralized nature of the Azure AD Verifiable Credentials system, there is no application user provisioning lifecycle.
 
-   * Do not store personally data claims returned in the VC token.
+   * Do not personally store data claims returned in the VC token.
```

Comment:
this needs to be reverted. store should be before personally

Reply:
should be do not store personal information data claims returned in the VC token

Reply:
This was fixed in a PR merged last week. I've resolved the merge conflict in favor of "Do not store personal data claims..."